### ISSUE TITLE
Only Inference script of Yolo-v8-obb.onnx. Cannot afford to have full Ultralytics bcz only have onnx-runtine and no Torch.

### ISSUE THREAD

**Issue by GISScience** [ultralytics/ultralytics#22139]
### Search before asking

- [x] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/orgs/ultralytics/discussions) and found no similar questions.

### Question

Only Inference script of Yolo-v8-obb.onnx. Cannot afford to have full Ultralytics bcz only have onnx-runtine and no Torch.
I cannot install the full Ultralytics package because there is no torch, and due to constraints, i cannot install it.
I have trained a model using Ultratrics and working fine.
But for inference a different machine, I only want inference on ONNX. 
How to run Yolo-v8-obb.onnx without full ultraltics.
Can you point to only the inference file of Yolo-v8-obb.onnx?

### Additional

I cannot install the full Ultralytics package because there is no torch, and due to constraints, i cannot install it.
usingh python 3.9 and i have only
numpy                               1.23.5
onnxruntime-gpu                     1.11.1
opencv-python                       4.5.1.48

**Comment by UltralyticsAssistant** [comment]
👋 Hello @GISScience, thank you for your interest in Ultralytics 🚀! This is an automated response to help you get started—an Ultralytics engineer will also assist you here soon.

We recommend a visit to the [Docs](https://docs.ultralytics.com/) for new users where you can find many [Python](https://docs.ultralytics.com/usage/python/) and [CLI](https://docs.ultralytics.com/usage/cli/) usage examples and where many of the most common questions may already be answered.

If this is a 🐛 Bug Report, please provide a [minimum reproducible example](https://docs.ultralytics.com/help/minimum-reproducible-example/) to help us debug it.

If this is a custom training ❓ Question, please provide as much information as possible, including dataset image examples and training logs, and verify you are following our [Tips for Best Training Results](https://docs.ultralytics.com/guides/model-training-tips/).

For your ONNX inference-only question, to help us assist you faster, please share:
- Your export details (command used to create the .onnx, opset, dynamic/static shapes)
- A small sample image and the exact onnxruntime code you attempted (if any)
- Any error messages or unexpected outputs (logs/screenshots)
- System info: OS, GPU/CPU, onnxruntime-gpu version, OpenCV version, Python version

Join the Ultralytics community where it suits you best. For real-time chat, head to [Discord](https://discord.com/invite/ultralytics) 🎧. Prefer in-depth discussions? Check out [Discourse](https://community.ultralytics.com/). Or dive into threads on our [Subreddit](https://www.reddit.com/r/Ultralytics/) to share knowledge with the community.

## Upgrade

Upgrade to the latest `ultralytics` package including all [requirements](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) in a [Python>=3.8](https://www.python.org/) environment with [PyTorch>=1.8](https://pytorch.org/get-started/locally/) to verify your issue is not already resolved in the latest version:

```bash
pip install -U ultralytics
```

## Environments

YOLO may be run in any of the following up-to-date verified environments (with all dependencies including [CUDA](https://developer.nvidia.com/cuda-zone)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) and [PyTorch](https://pytorch.org/) preinstalled):

- **Notebooks** with free GPU: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** Deep Learning VM. See [GCP Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** Deep Learning AMI. See [AWS Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. See [Docker Quickstart Guide](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Status

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml/badge.svg" alt="Ultralytics CI"></a>

If this badge is green, all [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yml?query=event%3Aschedule) tests are currently passing. CI tests verify correct operation of all YOLO [Modes](https://docs.ultralytics.com/modes/) and [Tasks](https://docs.ultralytics.com/tasks/) on macOS, Windows, and Ubuntu every 24 hours and on every commit.

**Comment by Y-T-G** [comment]
You can search GitHub. There are several repos.

https://github.com/search?q=yolov8+obb+onnx+language%3APython&type=code

**Comment by GISScience** [comment]
can you please help .
Which ultralytics version uses onnx-runtime = 1.11.1? I have to export yolov8-obb.pt to yolov8-obb.onnx only with onnx-runtime = 1.11.1

**Comment by Y-T-G** [comment]
Ultralytics always uses latest ONNX Runtime. It's not fixed. As long as ONNX IR version matches, it should work fine with latest ONNX Runtime.
